packages/subcommands/src/lib/Subcommand.ts

```diff
@@ -263,6 +259,15 @@ export class Subcommand<PreParseReturn extends Args = Args, O extends Subcommand
 
 export interface SubcommandOptions extends Command.Options {
 	subcommands?: SubcommandMappingArray;
+	/**
+	 * Whether to add aliases for subcommands with dashes in them
+	 *
+	 * Enabling this option will add aliases for both **message** subcommands *and* **chat input** subcommands
```

Comment:
You should only add aliases for entries that have message mappings. 

Also this doesn't handle group mappings :)